packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -23,6 +23,27 @@ export interface ClusterProps {
    * The VPC where your ECS instances will be running or your ENIs will be deployed
    */
   readonly vpc: ec2.IVpc;
+
+  /**
+   * The service discovery namespace created in this cluster
+   *
+   * @default none
+   */
+  readonly defaultCloudMapNamespace?: NamespaceOptions;
+
+  /**
+   * The name of the autoscaling group to be added
+   *
+   * @default none
+   */
+  readonly autoscalingGroupName?: string;
```

Comment:
Remove this prop, I don't think this is needed, we can just use something like `"Default"` as the default name.

Reply:
Removed and defaulted to 'DefaultAutoScalingGroup'